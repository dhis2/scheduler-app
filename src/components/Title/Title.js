import React from 'react'
import { string, number } from 'prop-types'

const Title = ({ children, priority }) => {
    const Heading = `h${priority}`

    return <Heading>{children}</Heading>
}

Title.defaultProps = {
    priority: 1,
}

Title.propTypes = {
    children: string.isRequired,
    priority: number,
}

export default Title
