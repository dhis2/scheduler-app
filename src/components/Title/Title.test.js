import React from 'react'
import { shallow } from 'enzyme'
import Title from './Title'

describe('<Title>', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Title>Text</Title>)

        expect(wrapper).toMatchSnapshot()
    })
})
