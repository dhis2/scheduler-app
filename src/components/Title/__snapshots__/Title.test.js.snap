// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title> renders correctly 1`] = `
<h1>
  Text
</h1>
`;
